var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -86,6 +86,9 @@ class Rocblas(CMakePackage):
     ]:
         depends_on(f"rocm-openmp-extras@{ver}", type="test", when=f"@{ver}")
 
+    for ver in ["6.2.0", "6.2.1"]:
+        depends_on(f"rocm-smi-lib@{ver}", type="test", when=f"@{ver}")
```

Comment:
Interesting. What's the error that this fixes?

Reply:
A ROCm-smi dependecy was added in 6.2 for client test: https://github.com/ROCm/rocBLAS/blob/rocm-6.2.0/clients/CMakeLists.txt#L163

This is the error:
```
CMake Error at /dockerx/spack/opt/spack/linux-ubuntu22.04-skylake_avx512/gcc-11.4.0/cmake-3.30.2-p4bgm3sa3x77w6z3l3xztmufc4cdmttl/share/cmake-3.30/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  Could NOT find ROCmSMI (missing: ROCM_SMI_LIBRARY ROCM_SMI_ROOT)
Call Stack (most recent call first):
  /dockerx/spack/opt/spack/linux-ubuntu22.04-skylake_avx512/gcc-11.4.0/cmake-3.30.2-p4bgm3sa3x77w6z3l3xztmufc4cdmttl/share/cmake-3.30/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
  /tmp/root/spack-stage/spack-stage-rocblas-6.2.0-dg5qqtgfq55x7seag22artrnkiyaz7tz/spack-build-dg5qqtg/virtualenv/lib/python3.11/site-packages/Tensile/Source/cmake/FindROCmSMI.cmake:46 (find_package_handle_standard_args)
  clients/CMakeLists.txt:165 (find_package)
```